src/Orleans/Core/ClientBuilder.cs

```diff
@@ -79,6 +80,8 @@ private static void AddBasicServices(IServiceCollection services, ClientConfigur
             services.AddFromExisting<IInternalGrainFactory, GrainFactory>();
             services.AddFromExisting<IGrainReferenceConverter, GrainFactory>();
             services.AddSingleton<ClientProviderRuntime>();
+            services.AddLogging();
+            services.AddSingleton<ILoggerFactory>(sp=> sp.GetRequiredService<ILoggerFactory>().AddConsole());
```

Comment:
does this mean we're always logging to the console?

Reply:
LogFactory is a singleton, I think if user configure their own logFacory, it overrides the default one. Is my understanding of DI correct?

Reply:
hmmm, this doesn't seem right...

Reply:
this is outdated code, I changed it to just services.AddLogging() in latest code. 

in 1.3 configure logging is not as easy as 2.0. That's why this weird code is there.

Reply:
Yes, AddLogging is fine, what is not is AddConsole. That's the user that needs to configure it however he wants